lua/cmake-tools/executors/terminal.lua

```diff
@@ -170,7 +171,7 @@ function terminal.send_data_to_terminal(buffer_idx, cmd, opts)
     -- do nothing
   end
 
-  if opts and not (opts.focus_on_launch_terminal or opts.focus_on_main_terminal) then
+  if opts and (opts.focus_on_launch_terminal or opts.focus_on_main_terminal) then
```

Comment:
Why did you remove the `not`?

Reply:
Slight bug here:

Sometimes there is a problem with buffers that are already open, which we do not want to focus on... like lets say we switch to nerdtree, and switch back to our `.cpp` file, and run `:CMake<...>` then it focuses on the nerdtree if that buffer is open... which can be annoying. So, a more robust system of detecting/switching buffers is needed like the buffer name/number and detection prior to `wincmd 'p'`.

Reply:
I modified this logic completely now 👀